### ISSUE TITLE
Device settings profiles.

### ISSUE THREAD

**Issue by BoB3R** [Map-A-Droid/MAD#859]
Hello,

There were few events that starts from 1 PM and reset quests. I've got 40+ devices and a lot of walkers, so when I want to scan the quests again after the night I need to change all the walkers to one that got quest area (that is the easy part). Issue is when I want to go back to old setup, I was using curl to do it but I always miss a new one or can make mistake and need to confirm with the screens. It would be cool to have profiles for device - walker, so I could save the current setup and restore it by a press of a button.

tl;dr

Profiles to save current setting for device to walker setup.

**Comment by dkmur** [comment]
can just have quest finish at 5pm. At 1pm delete the stuff from trs_quest you want rescanned and pply settings

**Comment by BoB3R** [comment]
It's not about that. I got like 30 walkers, they can be changed quickly in bulk to other walker to scan quests at 1 PM. The issue is about reverting it, if I want all the walker with raid and iv scanning back to normal. I need to change them one by one.

**Comment by Expl0dingBanana** [comment]
You can use the API parameter `link_disp_field` to pull the device /  walker configuration.  Save to a JSON and curl to re-implement at the end.  It should be a simple loop to perform these steps.
